packages/firestore/src/remote/remote_store.ts

```diff
@@ -236,8 +178,8 @@ export class RemoteStore {
    */
   disableNetwork(): Promise<void> {
     this.disableNetworkInternal();
-    // Set the OnlineState to failed so get()'s return from cache, etc.
-    this.updateOnlineState(OnlineState.Failed);
+    // Set the OnlineState to Offline so get()'s return from cache, etc.
```

Comment:
Don't use apostrophes for making plurals. (it should be "get()s" or just "gets").